SQLTask/Task6.sql

```diff
@@ -0,0 +1,21 @@
+USE DB_Bank
+
+SELECT Clients.FullName,
+	AVG(TotalBalance.SumBalance) AS FullBalance,
+	IIF(SUM(Cards.CardBalance) IS NULL, 
+		0, 
+		SUM(Cards.CardBalance))
+	 AS CardsSum,
+	IIF(SUM(Cards.CardBalance) IS NULL, 
+		AVG(TotalBalance.SumBalance), 
+		AVG(TotalBalance.SumBalance) - SUM(Cards.CardBalance))
+	 AS AvailableMoney
+FROM (SELECT Clients.FullName, 

```

Comment:
This whole query for task 6 can be made even, even simpler

Reply:
I still don't like your implementation, but I will accept this query. This is working, but performance can be better